src/extension/geoserver-layer-filter-extension/pom.xml

```diff
@@ -14,7 +14,7 @@
     <parent>
         <groupId>org.geoserver</groupId>
         <artifactId>extension</artifactId>
-        <version>2.7.0</version>
+        <version>2.8.0</version>
```

Comment:
Would prefer to use our extensions module - will look at whether we can do this easily separately